### PR TITLE
adding Komyu to the services page

### PR DESCRIPTION
**Pull request by OliveCannon**
Adding Komyu to the services page

### CODE REVIEW

File: content/en/community/services.md [hunk 1]
```diff
@@ -36,11 +36,17 @@ We do our best to ensure that the information on this page is accurate and helpf
...
         </td>
+         </tr>
+     <tr>
+        <td><img src="/images/logos/komyu.png" alt="Komyu"></td>
+        <td>Tools</td>
+        <td class="title">Komyu is a consulting and training company based in France that specializes in setting up Massive Cooperation worldwide. We believe that the world's challenges require the ability to work together in large groups, utilizing Inner Source, Open Source, well-engineered communities, and effective leadership. Our approach involves working on multiple levels, including corporate strategy, program tactics, operations, and individual leadership, to deploy Inner Source in both its operational dimensions (e.g. legal, tools, governance) and its human factors (e.g. sponsors, contributors, users). With global intervention capabilities, we aim to facilitate the adoption of Inner Source and foster collaboration on a massive scale.   <a href="https://www.komyu.io/">Click here to learn more</a> or <a href="https://www.komyu.io/#contact">contact Komyu</a> directly.
```
Comment (spier) [inline content/en/community/services.md]:
```suggestion
        <td class="title">Komyu is a consulting and training company based in France that specializes in setting up Massive Cooperation worldwide. We believe that the world's challenges require the ability to work together in large groups, utilizing InnerSource, Open Source, well-engineered communities, and effective leadership. Our approach involves working on multiple levels, including corporate strategy, program tactics, operations, and individual leadership, to deploy InnerSource in both its operational dimensions (e.g. legal, tools, governance) and its human factors (e.g. sponsors, contributors, users). With global intervention capabilities, we aim to facilitate the adoption of InnerSource and foster collaboration on a massive scale.   <a href="https://www.komyu.io/">Click here to learn more</a> or <a href="https://www.komyu.io/#contact">contact Komyu</a> directly.
```

suggesting to stick to the way that we spell `InnerSource` in most other places

File: content/en/community/services.md [hunk 2]
```diff
@@ -36,11 +36,17 @@ We do our best to ensure that the information on this page is accurate and helpf
...
+        <td class="title">Promyze supports InnerSource adoption by providing a knowledge-sharing platform for software developers, to create and share best coding practices. The tool is fully integrated in the developer ecosystem. Each team can define their best practices and choose to make them publicly available to other teams within the company. Teams can retrieve knowledge relevant to their context or can suggest improvements and ask questions. This boosts knowledge sharing and increases source code uniformity within a company.   <a href="https://promyze.com/innersource-understand-in-5-minutes/">Click here to learn more</a> or <a href="https://ze.com/contact">contact Promyze</a> directly.
```
Comment (spier) [inline content/en/community/services.md]:
```suggestion
        <td class="title">Promyze supports InnerSource adoption by providing a knowledge-sharing platform for software developers, to create and share best coding practices. The tool is fully integrated in the developer ecosystem. Each team can define their best practices and choose to make them publicly available to other teams within the company. Teams can retrieve knowledge relevant to their context or can suggest improvements and ask questions. This boosts knowledge sharing and increases source code uniformity within a company.   <a href="https://promyze.com/innersource-understand-in-5-minutes/">Click here to learn more</a> or <a href="https://promyze.com/contact">contact Promyze</a> directly.
```

This looks like it was changed by accident?

**Review by spier — DISMISSED** [review]
Made changes to the spelling of `InnerSource`. Also fixed something that looks like a mistake. Please confirm.

**Review by andrea-kyurchiev — APPROVED** [review]

**Comment by tfroment** [comment]
Hi, thank you for adding Komyu @andrea-kyurchiev @spier @OliveCannon : I submitted a minor set of improvements through https://github.com/InnerSourceCommons/innersourcecommons.org/pull/537

**Pull request merged**